### PR TITLE
Include a compilation timestamp in library metadata and sitemap

### LINKED ISSUE #468: Include compilation timestamp in library metadata
In order to render the sitemap we need to know when a library was last updated. The generated library metadata should contain a timestamp with the time the library was compiled.

### PR DESCRIPTION
**Pull request by dialelo**
Closes #468 

- [x] Merge 47deg/github4s#28 and release a snapshot
- [x] Ignore contributions without authors
- [x] Publish the new version of the content

@raulraja please review, I need to release a new version of the compiler and compiler plugin and open pull requests to update the content too so don't mind the failing build.

### CODE REVIEW

**Comment by raulraja** [comment]
LGTM!

**Comment by dialelo** [comment]
Don't merge yet, I need to publish the content first.

File: compiler/src/main/scala/com/fortysevendeg/exercises/compiler/TreeGen.scala [hunk 1]
```diff
@@ -87,8 +87,13 @@ case class TreeGen[U <: Universe](
...
   def makeLibrary(
-    name: String, description: String, color: Option[String],
-    sectionTerms: List[TermName], owner: String, repository: String
+    name:         String,
+    description:  String,
+    color:        Option[String],
+    sectionTerms: List[TermName],
+    owner:        String,
+    repository:   String,
+    timestamp:    String
```
Comment (andyscott) [inline compiler/src/main/scala/com/fortysevendeg/exercises/compiler/TreeGen.scala]: Should we use `Long` or another numeric type?

Comment (dialelo) [inline compiler/src/main/scala/com/fortysevendeg/exercises/compiler/TreeGen.scala]: We just need a string timestamp so i think it's overkill. We've used strings for contribution timestamps too.

**Comment by dialelo** [comment]
@FPerezP this changes are ready to merge, please take a look!

**Comment by FPerezP** [comment]
Looks good! Thanks!

**Pull request merged**